### DISCUSSION TITLE
韩语的语音转文本时出错了

### DISCUSSION THREAD

**Discussion by lx277856602** [ZuodaoTech/everyone-can-use-english#1130] [General]
首次导入一个韩语音频能够正常转文本，在导入第二个语音文字时转化失败，把第1个音频删除掉再转第2个也不行。出错信息如图所示
![error](https://github.com/user-attachments/assets/030aa344-812e-4488-96cf-f642ddcbab03)，请帮忙看看

**Comment by an-lee** [comment]
韩语确实有问题，得等上游依赖修复了 https://github.com/echogarden-project/echogarden/issues/57
